bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/DefaultSystemChannelTypeProvider.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.internal;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+
+import org.eclipse.smarthome.core.thing.type.ChannelType;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+import org.eclipse.smarthome.core.thing.type.SystemChannelTypeProvider;
+
+/**
+ * Implementation providing default system wide channel types
+ * 
+ * @author Ivan Iliev - Initial Contribution
+ * 
+ */
+public class DefaultSystemChannelTypeProvider implements SystemChannelTypeProvider {
+
+    /**
+     * Signal strength default system wide {@link ChannelType}. Represents signal strength of a device as a number
+     * with values 0, 1, 2, 3 or 4, 0 being worst strength and 4 being best strength.
+     */
+    private static final ChannelType SYSTEM_CHANNEL_SIGNAL_STRENGTH = new ChannelType(new ChannelTypeUID(
```

Comment:
Do you mean putting the ChannelTypeUID into an exported constants class? I think that makes sense, especially for dynamic ThingTypeProviders. But we should not move the whole ChannelTypeDefinition into the constants, because they can be retrieved via the getSystemChannelTypes() method.